src/main/java/teammates/ui/controller/Action.java

```diff
@@ -576,9 +576,9 @@ private boolean isMasqueradeModeRequested(AccountAttributes loggedInUser, String
                && loggedInUser.googleId != null
                && !loggedInUser.googleId.equals(requestedUserId);
     }
-    
+
     // ===================== Utility methods used by some child classes========
-    
+
```

Comment:
@Neurrone Earlier i did changes. So after that i reverted back to original file only.Please take a look at this image. Its same as master.
![temp](https://cloud.githubusercontent.com/assets/12005551/22465153/46556a58-e7e0-11e6-9e61-834583066aca.png)